### ISSUE TITLE
Developer documentation: Update needed for iPython with Anaconda 1.8.0

### ISSUE THREAD

**Issue by willingc** [cuthbertLab/music21#38]
The current version of Anaconda (1.8.0) does not support "conda pip" commands. Please let me know if you want me to add documentation for installation with Anaconda (1.8.0) and I will file a PR. Ultimately, it would be helpful to have music21 available through conda.

```
carol 10:21 PM -bash ~ ]conda pip music21
ERROR:
The "conda pip" command has been removed from conda (as of version 1.8) for
the following reasons:
  * users get the wrong impression that you *must* use conda pip (instead
    of simply pip) when using Anaconda
  * there should only be one preferred way to build packages, and that is
    the conda build command
  * the command did too many things at once, i.e. build a package and
    then also install it
  * the command is Python centric, whereas conda (from a package management
    perspective) is Python agnostic
  * packages created with conda pip are not robust, i.e. they will maybe
    not work on other people's systems

In short:
  * use "conda build" if you want to build a conda package
  * use "conda install" if you want to install something
  * use "pip" if you want to install something that is on PyPI for which there
    isn't a conda package.

carol 10:21 PM -bash ~ ]conda install music21
Error: No packages found matching: music21
carol 10:22 PM -bash ~ ]pip install music21
```
